client/src/components/MapMarker/MyLocationMarker.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Text,
+  View,
+} from 'react-native';
+import { Marker } from 'react-native-maps';
+
+import styles from './styles';
+
+
+const ANCHOR = { x: 0.5, y: 0.5 };
+
+const MyLocationMarker = ({ myPosition }) => {
+  // const rotate = (typeof myPosition.heading === 'number' && myPosition.heading >= 0) ?
+  // `${myPosition.heading}deg` : null; //disabled due to heading bug in library
+  const rotate = null; // dont draw heading
+  if (myPosition == null) return null;
+  if (!myPosition.latitude && myPosition.longitude) return null;
+
+
+  return (
+    <Marker
+      anchor={ANCHOR}
+      style={styles.mapMarker}
+      coordinate={myPosition}
+    >
+      <View style={styles.myMarkercontainer}>
+        <View style={styles.markerHalo} />
+        {rotate &&
```

Comment:
please remove this since it is unreachable - if you add it back in don't forget to cover it in the tests

Reply:
whys it unreachable?

Reply:
we talked about this. If you're going to leave it in please at least sort the indentation over the next few lines